packages/shared/src/lib/query.ts

```diff
@@ -15,7 +15,7 @@ import { SharedFeedPage } from '../components/utilities';
 
 export enum OtherFeedPage {
   Tag = 'tag',
-  Squad = 'squad',
```

Comment:
Was non-existent. After checking, the value in the `feedName` context is `squads`.